### ISSUE TITLE
Allow disabling Wi-Fi scan throttling outside Developer options

### ISSUE THREAD

**Issue by AtmosphericIgnition** [GrapheneOS/os_issue_tracker#4212]
As far as I'm aware, GrapheneOS plans to eventually ship support for using a network location service provider. Collecting data for open services like [BeaconDB](https://beacondb.net/) requires using a dedicated app for collecting the data. The quality and quantity of data collected can be improved by disabling Wi-Fi scan throttling in Developer options.
Since the GrapheneOS team has [encouraged](https://grapheneos.social/@GrapheneOS/112759424689620738) submitting data to open location services, it would be nice if GrapheneOS made this effort easier by allowing the disabling of Wi-Fi scan throttling without having Developer options enabled. Some network analysis apps can also benefit from this setting being disabled. As far as I understand, the only downside of disabling Wi-Fi scan throttling is increased battery usage when an app is actively scanning for networks.

I wanted to link a blog from Mozilla, which talked about how Wi-Fi scan throttling was impacting their MLS data collection efforts, but I haven't been able to find it.

**Comment by secretmango** [comment]
Yes agreed. I read the thread on Matrix, GrapheneOS had a smaller community so contributing can mean your movements are revealed a bit.

But contributing is totally opt-in.

Having a button to
- disable wifi scan throttling
- disable data saver if that impacts it
- disable energy saver
- pretend the device is charging (?)

Could help improve the data that is gathered. I do this manually, and it is not great.
